tests/e2e/local/common.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+
+# Copyright 2018 Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Common utilities for Istio local installation.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Install Curl on Ubuntu platform
+function install_curl_ubuntu() {
```

Comment:
I love moving all these to functions so they can be reused.

However, could you separate linux/macos utils into different files for better readability?

Reply:
@hklai 

> could you separate linux/macos utils into different files for better readability?

Make sense, it's fixed now. Please take a look when you have chance :) THANKS!